www/js/survey/enketo/answer.js

```diff
@@ -1,11 +1,9 @@
 import angular from 'angular';
 import MessageFormat from 'messageformat';
+import { getConfig } from '../../config/dynamicConfig';
 
-angular.module('emission.survey.enketo.answer', [
-  'ionic',
-  'emission.config.dynamic',
-])
-.factory('EnketoSurveyAnswer', function($http, DynamicConfig,) {
+angular.module('emission.survey.enketo.answer', ['ionic'])
+.factory('EnketoSurveyAnswer', function($http) {
```

Comment:
I don't think this file actually uses `ionic` and it could be removed as a dependency here.

But unfortunately, we are not fully done with `ionic`. After this PR we don't use any of the `ion-*` HTML elements. But many of the services that haven't been rewritten do still use `$ionicPopup` and `$ionicPlatform`.